tests/pipelines/kitctl/template.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: pipeline-template
```

Comment:
If we think of this in terms of a library, this name will conflict. Maybe we can come up with naming scheme like folder == pipeline name, e.g. `kitctl` here?

Reply:
This is a template and they should not use `pipeline-template` as the pipeline name for the final pipeline. I think it's the customer's job to make sure that all their pipelines are uniquely named according to what they want to do. For example, if i am extending this template to test ETCD qps performance, I should name the pipeline `ETCD-QPS-TEST`.